src/e2e/helpers/mobile/gestures/gesture.ts

```diff
@@ -0,0 +1,37 @@
+import { Direction } from '../../../../types'
+import { Browser } from 'webdriverio'
+
+export interface GestureOptions {
+  xStart?: number,
+  yStart?: number,
+  segmentLength?: number,
+}
+
+interface Action<T = any> {
+  action: string,
+  options?: T,
+}
+
+/** Apply gesture action for the given path. */
+const gesture = async (browser: Browser<'async'>, path: Direction[], { xStart = 70, yStart = 300, segmentLength = 70 }: GestureOptions = {}) => {
+
+  const moveActions = path.reduce<Action[]>((acc, cur) => {
```

Comment:
You can type the `reduce` call: `path.reduce<Action[]>` and then all the arguments will be typed correctly.

Reply:
Good, thank you!